src/app/scripts/cac/pages/cac-pages-home.js

```diff
@@ -129,14 +130,24 @@ CAC.Pages.Home = (function ($, ModeOptions,  MapControl, Modal, Templates, UserP
             });
         }
 
-        // TODO: re-enable loading settings from user preferences
-        // once routing figured out. Currently there is no way to go back
-        // to the home page, so if there is an origin and destination in
-        // preferences, the app will jump directly to the map page with no way back.
-        $(document).ready(directionsControl.setFromUserPreferences());
-    };
+        $(options.selectors.tabControl).on('click', options.selectors.tabControlLink, function (event) {
+            var tabId = $(this).data('tab-id');
+            if (tabId === tabControl.TABS.EXPLORE) {
+                event.preventDefault();
+                event.stopPropagation();
 
-    return Home;
+                tabControl.setTab(tabControl.TABS.EXPLORE);
+            }
+        });
+        $(options.selectors.homeLink).on('click', function (event) {
+            event.preventDefault();
+            event.stopPropagation();
+
+            tabControl.setTab(tabControl.TABS.HOME);
+        });
+
+        $(document).ready(directionsControl.setFromUserPreferences());
```

Comment:
Might want to leave a TODO in here about user preferences going away/needing refactor still